lib/hammer_cli_katello/output/fields.rb

```diff
@@ -10,6 +10,9 @@ class ChecksumFilePair < ::Fields::Field
       class Dependency < ::Fields::Field
       end
 
+      class Array < ::Fields::Field
```

Comment:
The difference is in how it's formatted https://github.com/Katello/hammer-cli-katello/pull/353/files#diff-9ae5ae9df1629bf9fc24b04247dabfcdR29
`List` is printed on one line, items separated by commas, while `Array` puts each item on it's own line. So it's a cosmetic thing. We could probably just keep both as list and switch with a field parameter too.